doc/workers.rst

```diff
@@ -25,9 +27,33 @@ Available workers:
   ``worker_type: aws``, and for more details on the setup and configuration,
   see below.
 
+.. _conda_env_worker:
+
+Running submissions with a worker using Conda environment
+---------------------------------------------------------
+
+As previously mentioned, the :class:`ramp_engine.local.CondaEnvWorker` workers
+rely on Conda environment to execute the submission.
+
+For instance, if you would like to run the `Iris challenge
+<https://github.com/ramp-kits/iris>`_, the starting kit requires numpy, pandas,
+and scikit-learn. Thus, the environment used to run the initial submission
+should have these libraries installed.
+
+Subsequently, you can create such environment and use it in the event
+configuration::
 
-Running a submissions on Amazon Web Services (AWS)
---------------------------------------------------
+      conda create --name ramp-iris numpy pandas scikit-learn
```

Comment:
I think ramp-workflow is missing here? 
So it will not be a one liner to have the environment

Reply:
true